apps/web/src/components/score/ScoreVisual.tsx

```diff
@@ -0,0 +1,90 @@
+import ScoreProfile from './ScoreProfile';
+import ScoreInfo from './ScoreInfo';
+import { UserScore } from 'types/Score';
+import { User } from 'types/User';
+import { Text, Image, Box, useMediaQuery, Flex } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { mobileData, defaultData } from './ScoreConfig';
+import ScoreColored from './ScoreColored';
+import ScoreColoredMobile from './ScoreColoredMobile';
+
+type Props = {
+  size?: number;
+  user?: User;
+  userScore: UserScore;
+};
+
+export default function ScoreVisual({ size = 1, user, userScore }: Props) {
+  const [isLargerThan768] = useMediaQuery('(min-width: 768px)');
```

Comment:
improve this name: isLargerThan768, it doens't scale
change to isDesktop or isMobile